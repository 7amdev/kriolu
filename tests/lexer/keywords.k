e ou 
ka klasi keli
si sinou super
falsu funson
di divolvi
ti timenti
imprimi
verdadi
mimoria
nulo