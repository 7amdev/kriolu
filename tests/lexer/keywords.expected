 1 <e>       'e' 
 1 <ou>      'ou' 
 2 <ka>      'ka' 
 2 <klasi>   'klasi' 
 2 <keli>    'keli' 
 3 <si>      'si' 
 3 <sinou>   'sinou' 
 3 <super>   'super' 
 4 <falsu>   'falsu' 
 4 <funson>  'funson' 
 5 <di>      'di' 
 5 <divolvi> 'divolvi' 
 6 <ti>      'ti' 
 6 <timenti> 'timenti' 
 7 <imprimi> 'imprimi' 
 8 <verdadi> 'verdadi'
 9 <mimoria> 'mimoria'
 10 <nulo>   'nulo'